docs/content/getting-started/setup.md

```diff
@@ -66,7 +66,7 @@ Click the preferred option to proceed to option-specific docs.
 
 With this method, Docker will run inside a VM in VirtualBox.
 
-1. Install VirtualBox
+1. Install VirtualBox v5.2.32
```

Comment:
```suggestion
1. Install VirtualBox
```

Reply:
This was a conscious change. Versions are not shown anywhere on the page otherwise.

Now they do:

![image](https://user-images.githubusercontent.com/1205005/76670255-d5604f00-654c-11ea-82e0-22f8c7f234e4.png)

Reply:
Yeah I know, that's exactly why I propose not to write it again in another spot